docs-ui/stories/components/areaChart.stories.js

```diff
@@ -1,6 +1,6 @@
 import styled from '@emotion/styled';
 
-import AreaChart from 'sentry/components/charts/areaChart';
+import {AreaChart} from 'sentry/components/charts/areaChart';
```

Comment:
Why change from `export default`?

Reply:
I'm not sure if we've already decided if we're going to use named exports or if we're going to stick with default exports, but since that's not decided yet and I see  benefits using named exports, I've updated it here, but I can revert if you want